envs/test/nginx/nginx.conf

```
@@ -0,0 +1,20 @@
+server{
```

тут не хватает пробела)
```suggestion
server {
```